src/agent/src/image.rs

```diff
@@ -66,26 +75,41 @@ impl ImageService {
         if !guest_pause_bundle.exists() {
             bail!("Pause image not present in rootfs");
         }
+        let guest_pause_config = guest_pause_bundle.join(CONFIG_JSON);
 
         info!(sl(), "use guest pause image cid {:?}", cid);
         let pause_bundle = Path::new(CONTAINER_BASE).join(cid).join(target_subpath);
         let pause_rootfs = pause_bundle.join("rootfs");
         fs::create_dir_all(&pause_rootfs)?;
-
-        let copy_if_not_exists = |src: &Path, dst: &Path| -> Result<()> {
-            if !dst.exists() {
-                info!(sl(), "copying file {src:?} to {dst:?}");
-                fs::copy(src, dst)?;
-            }
-            Ok(())
-        };
-        copy_if_not_exists(
-            &guest_pause_bundle.join(CONFIG_JSON),
-            &pause_bundle.join(CONFIG_JSON),
-        )?;
+        info!(sl(), "pause_rootfs {:?}", pause_rootfs);
+        let image_oci = oci::Spec::load(guest_pause_config.to_str().ok_or_else(|| {
+            anyhow!(
+                "Invalid container image OCI config path {:?}",
+                guest_pause_config
+            )
+        })?)
+        .context("load image config file")?;
+
+        let image_oci_process = image_oci.process.ok_or_else(|| {
+            anyhow!("The image config file does not contain a process specification.")
```

Comment:
And here: "The guest pause image config does not contain a process specification. Please check the pause image" to be consistent with the message below when `args.len()` checker fails.